consumers.cabal

```diff
@@ -76,3 +78,25 @@ test-suite consumers-example
                       text,
                       text-show,
                       transformers
+
+executable consumers-test
+  hs-source-dirs:     example
```

Comment:
I'd also have this stuff live in the `test` directory.

Reply:
Actually, since the test suite is not part of this PR, just remove this section, it makes the CI fail.

Reply:
I forgot to add the test. It is there now.